templates/aws/mysql_rds.cfn.template.yml

```diff
@@ -29,7 +29,4 @@ Resources:
       VPCSecurityGroups:
         - !Ref SECURITYGROUPID
       EnablePerformanceInsights: true
-      CopyTagsToSnapshot: true
-      Tags: 
```

Comment:
As per your earlier feedback "If you just tag the stack with LansaVersion, all the resources will automatically be tagged. Remove all the individual resource tags" we will be tagging the stack and it will pass the tags to resources as well.